pr-data.csv

```diff
@@ -2625,20 +2625,20 @@ https://github.com/apache/opennlp,8209cd308ee6ff7e61ec0346f1799a32541ed80f,openn
-https://github.com/apache/openwebbeans,01245cf8ef0baaa9cd31342b0cdb075d314b6ef9,webbeans-tck,org.jboss.cdi.tck.tests.full.implementation.producer.method.lifecycle.ProducerMethodLifecycleTest.arquillianBeforeClass,ID,,,
-https://github.com/apache/openwebbeans,01245cf8ef0baaa9cd31342b0cdb075d314b6ef9,webbeans-tck,org.jboss.cdi.tck.tests.full.inheritance.specialization.qualifiers.SpecializingBeanQualifiersTest.arquillianBeforeClass,ID,,,
-https://github.com/apache/openwebbeans,424af4bb7285b806d3e3efe5020cd3e440e0a4ff,webbeans-tck,org.jboss.cdi.tck.tests.implementation.simple.lifecycle.SimpleBeanLifecycleTest.arquillianBeforeClass,ID,,,
+https://github.com/apache/openwebbeans,424af4bb7285b806d3e3efe5020cd3e440e0a4ff,webbeans-impl,org.apache.webbeans.test.unittests.specializes.logger.LoggerSpecializationTest.testSpecializedVersion,ID,,,
+https://github.com/apache/openwebbeans,424af4bb7285b806d3e3efe5020cd3e440e0a4ff,webbeans-jetty9,org.apache.webbeans.web.jetty9.test.JettyNormalScopeProxyFactoryTest.checkDeserialisation,ID,,,
+https://github.com/apache/openwebbeans,424af4bb7285b806d3e3efe5020cd3e440e0a4ff,webbeans-tomcat7,org.apache.webbeans.web.tomcat7.test.TomcatNormalScopeProxyFactoryTest.checkDeserialisation,ID,,,
+https://github.com/apache/openwebbeans,424af4bb7285b806d3e3efe5020cd3e440e0a4ff,webbeans-tck-jakart,org.jboss.cdi.tck.tests.extensions.configurators.observerMethod.ObserverMethodConfiguratorTest.notifyAcceptingConsumerNotified,ID,Deleted,,https://github.com/apache/openwebbeans/pull/51
+https://github.com/apache/openwebbeans,01245cf8ef0baaa9cd31342b0cdb075d314b6ef9,webbeans-tck,org.jboss.cdi.tck.tests.full.extensions.configurators.observerMethod.ObserverMethodConfiguratorTest.notifyAcceptingConsumerNotified,ID,Deleted,,https://github.com/apache/openwebbeans/pull/51
+https://github.com/apache/openwebbeans,01245cf8ef0baaa9cd31342b0cdb075d314b6ef9,webbeans-tck,org.jboss.cdi.tck.tests.full.implementation.producer.method.definition.ProducerMethodDefinitionTest.arquillianBeforeClass,ID,Deleted,,https://github.com/apache/openwebbeans/pull/51
+https://github.com/apache/openwebbeans,01245cf8ef0baaa9cd31342b0cdb075d314b6ef9,webbeans-tck,org.jboss.cdi.tck.tests.full.implementation.producer.method.lifecycle.ProducerMethodLifecycleTest.arquillianBeforeClass,ID,Deleted,,https://github.com/apache/openwebbeans/pull/51
+https://github.com/apache/openwebbeans,01245cf8ef0baaa9cd31342b0cdb075d314b6ef9,webbeans-tck,org.jboss.cdi.tck.tests.full.inheritance.specialization.qualifiers.SpecializingBeanQualifiersTest.arquillianBeforeClass,ID,Deleted,,https://github.com/apache/openwebbeans/pull/51
+https://github.com/apache/openwebbeans,424af4bb7285b806d3e3efe5020cd3e440e0a4ff,webbeans-tck,org.jboss.cdi.tck.tests.implementation.simple.lifecycle.SimpleBeanLifecycleTest.arquillianBeforeClass,ID,Deleted,,https://github.com/apache/openwebbeans/pull/51
```

Comment:
We discussed this in class, but I think they should not be marked as `Deleted`.

Can you split this PR? Let's make this PR only update the `DeveloperFixed`. And please raise another PR to remove these tests and explain the situation again in the PR description so that we can discuss with Darko.